packages/reporter/src/sessions/sessions-model.ts

```diff
@@ -7,19 +7,29 @@ export interface SessionProps extends InstrumentProps {
   testCurrentRetry: number
   sessionInfo: {
     id: string
-    data: Record<string, {cookies: number, localStorage: number}>
+    isGlobalSession: boolean
+    status: 'creating' | 'created' | 'restored' |'restored' | 'recreating' | 'recreated' | 'failed'
   }
 }
 
 export default class Session extends Instrument {
   @observable name: string
-  @observable data: SessionProps['sessionInfo']['data']
+  @observable status: string
+  @observable isGlobalSession: boolean = false
 
   constructor (props: SessionProps) {
     super(props)
-    const { id, data } = props.sessionInfo
+    const { sessionInfo } = props
```

Comment:
is this type safe?
```suggestion
    const { sessionInfo: { isGlobalSession, name, status } }= props
```

Reply:
Yes, but at the mercy of emitted log properties.  Updates seems good though 👍🏻